client/src/components/styles/_mixin.scss

```diff
@@ -19,3 +19,34 @@
     }
   }
 }
+@mixin flex_item($align,$justify){
+  display: flex;
+  align-items:  $align;
+  justify-content:$justify;
+}
+@mixin adaptiveness($max_width,$width,$padding,$padding-left){
+@media screen and (max-width: $max_width){
+.input_input {
+  width: $width;
```

Comment:
if u don't use mixin this part of code will take only 1 line. If u use the mixin, it is at least 4. What the purpose of this mixin?

Reply:
@mixin adaptiveness($max_width,$width,$padding,$padding-left){
@media screen and (max-width: $max_width){
.input_input {
  width: $width;
}
.li_component {
  padding: $padding;
}
.logo_component .jsgram {
    padding-left: $padding-left;
  }
}}
If there is no mixin media queries, one media query takes 7 lines. And I use media queries 3 times. So it will be 21 lines and with mixin it is 10 lines